IFC4x3/Properties/t/TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK/Documentation.md

```diff
@@ -1 +1,2 @@
-Indicates the stuck rate of tickets in the turnstile.
+Indicates the ratio of tickets being stuck or jammed in appliance or turnstile.
```

Comment:
Was this property added somewhere already? Don't see it in this PR.
```suggestion
Indicates the ratio of tickets being stuck or jammed in the appliance or turnstile.
```

Reply:
Yes it was in IFC door for turnstiles